docker/dbt/profiles.yml

```diff
@@ -0,0 +1,11 @@
+default:
```

Comment:
Can we add a this block to the top or bottom?
```
config:
    send_anonymous_usage_stats: False
```

That will prevent us from collecting anonymous usage stats about test runs.

Reply:
Added: https://github.com/fishtown-analytics/dbt-presto/pull/10/commits/ee8347268aaf51a9f26db7bdb87199c699c3be4d